ui/js/controllers/bugfiler.js

```diff
@@ -0,0 +1,224 @@
+
+            for(var i=0; i < omittedLeads.length; i++) {
+                if(summary[0].search(omittedLeads[i]) >= 0) {
+                    summary.shift();
+                }
+            }
+
+            $uibModalInstance.possibleFilename = summary[0].split("/").pop();
+
+            return [summary, $uibModalInstance.possibleFilename];
+        };
+
+        $scope.toggleFilerSummaryVisibility = function() {
+            $scope.isFilerSummaryVisible = !$scope.isFilerSummaryVisible;
+        };
+
+        $scope.isFilerSummaryVisible = false;
+
+        /*
+         *  Attempt to find a good product/component for this failure
+         */
+        $scope.findProduct = function() {
+            $scope.suggestedProducts = [];
+            var failurePath = $uibModalInstance.parsedSummary[0][0];
+            var failurePathRoot = failurePath.split("/")[0];
+
+            // Look up the product via the root of the failure's file path
+            if(thBugzillaProductObject[failurePathRoot]) {
+                $scope.suggestedProducts.push(thBugzillaProductObject[failurePathRoot][0]);
+            }
+
+            // Look up product suggestions via Bugzilla's api
+            var productSearch = $scope.productSearch;
+
+            if(productSearch) {
+                $http.get("https://bugzilla.mozilla.org/rest/prod_comp_search/" + productSearch + "?limit=5").then(function(request) {
+                    var data = request.data;
+                    $scope.suggestedProducts = [];
+                    for(var i = 0; i < data.products.length;i++) {
+                        if(data.products[i].product && data.products[i].component) {
+                            $scope.suggestedProducts.push(data.products[i].product + " :: " + data.products[i].component);
+                        }
+                    }
+                });
+            }
+        };
+
+        /*
+         *  This is called once intermittent.html's ng-repeat is finished to select the first product listed
+         */
+        $scope.focusProduct = function() {
+            $("#suggestedProducts").children(":first").children(":first").prop("checked", true);
+        };
+
+        /*
+         *  Same as clicking outside of the modal, but with a nice button-clicking feel...
+         */
+        $scope.cancelFiler = function() {
+            $uibModalInstance.dismiss('cancel');
+        };
+
+        /*
+         *  Actually send the gathered information to bugzilla.
+         */
+        $scope.submitFiler = function() {
+            var bugzillaRoot = "https://bugzilla-dev.allizom.org/"; // (prod is "https://bugzilla.mozilla.org/");
+            var summarystring = $scope.modalSummary;
+            var productString = "";
+            var componentString = "";
+            var isProductSelected = false;
+
+            $(':input','#modalForm').attr("disabled",true);
+
+            var productRadios = $("#modalForm input[name='productGroup'");
```

Comment:
Looks like this comment got missed.  Instead of using ``productRadios`` here, you can just reference ``$scope.selectedProduct`` to do your decision making.